data/systemd/snapd.manage

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+#
+# This script acts as an entry point for running snapd from snapd.snap on
+# a base snap that otherwise doesn't ship snapd itself. The entry point
+# arranges for appropriate system integration, in particular by creating
+# and starting systemd units for running snapd without any confinement.
+#
+# Arguments:
+# $1: start|stop
+# $2: prefix of the installed snapd snap (e.g. /snap/snapd/42/)
+#     (on a fedora base this might be e.g. /var/lib/snaps...)
+
+set -e
+
+CMD="$1"
+PREFIX="$2"
+
+UNITS="snapd.service snapd.autoimport.service snapd.core-fixup.service \
+       snapd.seeded.service snapd.snap-repair.service \
+       snapd.snap-repair.timer snapd.socket"
+ENABLE_UNITS="snapd.system-shutdown.service"
+
+# TODO: discuss ID_LIKE=ubuntu-core or ID_LIKE=snap-core ?
+if ! grep -q -E "ID(_LIKE)?=ubuntu-core" /etc/os-release && \
+        ! grep -q -E "ID(_LIKE)?=ubuntu-core" /usr/lib/os-release;
+    then
+        echo "cannot run this helper on a non ubuntu-core system"
+    exit 1
+fi
+
+case $CMD in
+    start)
+        echo "Running start action"
```

Comment:
Where are the logs from this script going to end up in?

Reply:
This will go into the systemd journal, i.e. systemctl status snapd.manage will tell (or whatever name we pick).